src/config/functions.ts

```diff
@@ -149,5 +137,12 @@ export function createProviderFunctions(
 				]);
 			},
 		},
+		unmanaged: {
+			get: () => {
+				return config.get('apiEndpoint').then(apiEndpoint => {
```

Comment:
This can be
```ts
get: () => config.get('apiEndpoint').then(_.negate)

Reply:
The typings aren't good enough yet for that to be allowed, so rather than doing lots of casting, I think the current version would be better.

Reply:
It's not the typings, I got confused about the _.negate signature - you use it for things like `_.negate(_.isNil)` to wrap a function and negate it's result, so it could be done as `_.negate(_.identity)` but I think that's probably less clear